pkg/imageutils/sys_context_test.go

```
@@ -171,16 +172,16 @@ MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuOSW8w==
 			expectCerts:   true,
 		},
 		{
-			name: "WithControllerConfig only - proxy",
+			name: "WithControllerConfig only - proxy in controllerconfig does not set DockerProxyURL",
 			controllerConfig: &mcfgv1.ControllerConfig{
 				Spec: mcfgv1.ControllerConfigSpec{
 					Proxy: &configv1.ProxyStatus{
 						HTTPSProxy: "https://proxy.example.com:3128",
```

golangci-lint (2.12.2)

[error] string `https://proxy.example.com:3128` has 7 occurrences, make it a constant

(goconst)